src/NServiceBus.CastleWindsor.Tests/app.config

```diff
@@ -4,7 +4,11 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Castle.Windsor" publicKeyToken="407dd0808d44fbdc" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.4.0.0" newVersion="3.4.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-3.4.0.0" newVersion="4.0.0.0" />
```

Comment:
It could be, the base version in the core project is 3.4.0 though so it probably doesn't make a difference. I'll update.